arlpy/uwapm.py

```diff
@@ -634,12 +876,18 @@ def run(self, env, task, debug=False, fname_base=None):
 
     def _bellhop(self, *args):
         try:
-            result = _proc.run(f'bellhop.exe {" ".join(list(args))}', 
-                        stderr=_proc.STDOUT, stdout=_proc.PIPE,
-                        shell=True)
-            if result.returncode == 127:
-                return False
-        except OSError:
+            for i in args:
+                if i.endswith(".env"):
+                    i.replace(".env", "")
```

Comment:
Why is this needed? We always call `_bellhop()` with `fname_base`, which does not contain the `.env` extension.